### ISSUE TITLE
Warning: Unknown mode: [object Object] Use --force to continue.

### ISSUE THREAD

**Issue by akinnee** [mazerte/grunt-jscpd#12]
Getting this:

```
Running "jscpd:finddupes" (jscpd) task
warn: File .cpd.yaml not found in current directory, or it is broken
info: Files search time: duration=21275ms
info: Scaning 464 files for duplicates...
>> Error: Unknown mode: [object Object]
Warning: Unknown mode: [object Object] Use --force to continue.
```

Here is my config:

```
grunt.config.set('jscpd', {
		finddupes: {
			path: './',
			files: 'app/**/*.js',
			exclude: [
				'**/node_modules/**',
				'**/compiled/**',
				'**/templates/**',
				'**/vendor/**',
				'**/*_compiled.js',
				'**/*Template.js'
			],
			output: 'tests/_duplicates-report.xml'
		}
	});
```

And I'm running the command `grunt jscpd`.

**Comment by glispajon** [comment]
I've got the same problem. Seems to be a massive difference in the package npm downloads now compared to the one a week or two ago which is very confusing since there hasn't been any activity in github for a while

**Comment by guy-mograbi-at-gigaspaces** [comment]
+1 me too. same issue

**Comment by vikash20186** [comment]
did you guys got any solution to the above problem? I am getting it too :(

**Comment by guy-mograbi-at-gigaspaces** [comment]
nope. and no response. the plugin is simply not working.. I was sure someone would post something.

**Comment by guy-mograbi-at-gigaspaces** [comment]
I found out that .html files might cause the problem. 
if I run the plugin on app/scripts - which contains only .js files - everything works great. 

This is still an issue that should be resolved since I cannot find copy-pastes in html - but at least I can find it in js files.

**Comment by skyl** [comment]
Agree that adding html causes the problem. But, I want to find duplicates in html, as well. However, I'm using straight jscpd with no grunt/gulp wrapper. So, this is a jscpd issue, not and issue with the grunt plugin here.

**Comment by Chris2011** [comment]
I had the same issue and with excluding **/*.html it worked.
